packages/logger/src/transports/storage.ts

```diff
@@ -444,6 +428,8 @@ export class StorageTransport implements ITransport {
      */
     extraLogs?: ExtraLogs;
   } = {}) {
+    // before query logs, prune logs
+    this._pruneLogs();
```

Comment:
doe the not await is by desige, if so, the comment should change and more cleaar